collectd-openstack/README.md

```diff
@@ -0,0 +1,300 @@
+# ![](./img/integrations_openstack.png) OpenStack
+
+Metadata associated with the OpenStack plugin for collectd can be found [here](https://github.com/signalfx/integrations/tree/release/collectd-openstack). The relevant code for the plugin can be found [here](https://github.com/signalfx/collectd-openstack).
+
+- [Description](#description)
+- [Requirements and Dependencies](#requirements-and-dependencies)
+- [Installation](#installation)
+- [Configuration](#configuration)
+- [Usage](#usage)
+- [Metrics](#metrics)
+- [License](#license)
+
+### DESCRIPTION
+
+This is the SignalFx OpenStack plugin. Follow these instructions to install the OpenStack plugin for collectd.
+
+The [openstack-collectd](https://github.com/signalfx/collectd-openstack) plugin collects metrics from OpenStack components by hitting various endpoints. This plugin covers the following components:
+
+* Nova (Compute)
+* Cinder (BlockStorge)
+* Neutron (Network)
+
+Reference for OpenStack [Monitoring](https://wiki.openstack.org/wiki/Operations/Monitoring).
+
+#### FEATURES
+
+#### Built-in dashboards
+
+
+- **HYPERVISOR**: Provides a high-level overview metrics for an OpenStack hypervisor.
```

Comment:
.. overview of metrics...  (Sorry I missed this the first time)

Reply:
corrected